src/Tizen.NUI.Scene3D/src/public/Controls/SceneView.cs

```diff
@@ -834,6 +834,36 @@ private Rotation GetSkyboxOrientation()
             return ret;
         }
 
+        /// <summary>
+        /// Callback when CornerRadius property changed.
+        /// </summary>
+        [Obsolete("Do not use this, that is deprecated in API13.")]
+        internal override void ApplyCornerRadius()
```

Comment:
Please do not revive this API. SceneView don't need `ApplyCornerRadius()` anymore. (Also, we cannot animate corner radius of sceneview before, actually)